pkg/storage/mongo/getter.go

```diff
@@ -1,18 +1,18 @@
 package mongo
 
 import (
-	"bytes"
 	"context"
-	"io/ioutil"
 	"strings"
 
 	"github.com/globalsign/mgo/bson"
+	"github.com/gomods/athens/pkg/errors"
 	"github.com/gomods/athens/pkg/storage"
 	opentracing "github.com/opentracing/opentracing-go"
 )
 
 // Get a specific version of a module
 func (s *ModuleStore) Get(module, vsn string) (*storage.Version, error) {
+	const op errors.Op = "mongo.Get"
```

Comment:
@marwan-at-work I'm really liking this standardized error handling. just thought you should know - not really related to this PR

🎉

Reply:
@arschles I appreciate you 🕺